plugins/arcgis/web-app/projects/main/src/lib/arc-admin/arc-admin.component.ts

```diff
@@ -69,14 +72,79 @@ export class ArcAdminComponent implements OnInit {
     this.config = config;
     this.configChangedNotifier.next();
   }
-
+// Visually blank but has underlying value in editConfig
   ngOnInit(): void {
-  }
+    this.attributesForm = this.fb.group({
```

Comment:
This should be populated with the config returned from the server.  You can move this definition to the constructor and then populate the config with values returned from `fetchArcConfig(...)`

Reply:
Okay, I think I did this correctly.